docs/Technical-Documentation/error-categories-guide.md

```diff
@@ -0,0 +1,81 @@
+# Error Categories Guide
```

Comment:
```suggestion
# Error Categories Guide

TDP checks submitted data files (and data therein) for errors by _first_ checking if the data file being uploaded is the correct [file type](https://github.com/raft-tech/TANF-app/pull/2725), _then_ proceeding with a sequence of checks designed to determine if the data in the file can be parsed, and if so, find any issues with the data values reported. 

This guide provides a high-level overview of these types of errors. For more specific details, see [TDP's parsing directory](https://github.com/raft-tech/TANF-app/tree/develop/tdrs-backend/tdpservice/parsers) for current pre- and post-parsing validation logic.
```

the file type link i added is the PR, but could be a link to KC if that merges first.